CONTRIBUTING.md

```diff
@@ -1,22 +1,49 @@
 ### Contributing to ECMAScript
-There are a number of ways to contribute to ECMAScript. How to actually contribute depends on what you want to accomplish. In many cases, you may be required to execute a CLA with Ecma. See the final section named *Signing the CLA* for more information on this.
+Contributors to ECMAScript and TC39 are expected to follow our [Code of Conduct](https://tc39.github.io/code-of-conduct/).
 
-Ecma TC39 accepts contributions from non-member individuals who have accepted the TC39 copyright and patent policies. Currently all ECMAScript related technical work is done by the TC39 RF TG (Royalty Free Task Group), for which the following IPR Policies apply:
+If you are not an Ecma member, any non-trivial contributions require signing a legal agreement with Ecma. See the section "Required Legal Agreements" below for details.
 
-  * [Ecma International RF Patent Policy](https://ecma-international.org/memento/Policies/Ecma_Royalty-Free_Patent_Policy_Extension_Option.htm)
-  * [Ecma International Software Copyright Policy](https://ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.htm) ([PDF](https://ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.pdf))
+#### Issues and Pull Requests
+For small changes to ECMAScript, you can contribute by filing an issue or a pull request against the current text of ECMA-262 standard in this repository.
+
+To file an issue, go to the ecma262 [issues page](https://github.com/tc39/ecma262/issues). From there, [examine](https://guides.github.com/features/issues/) in the existing issues to see if an issue already exists to track this issue. If so, add a comment to the existing issue; otherwise, [file a new issue](https://help.github.com/articles/creating-an-issue/) documenting the problem.
+
+To make a pull request (PR), [fork](https://help.github.com/articles/fork-a-repo/) the [ecma262](https://github.com/tc39/ecma262), apply changes to the spec.html, and upload it to your fork on GitHub, using the web interface to file a pull request. Locally, to see how your change renders in HTML, use the [ecmarkup](https://github.com/bterlson/ecmarkup) tool to convert spec.html into an actual HTML file.
+
+Issues and PRs in the ecma262 repository are appropriate for minor modifications to the existing specification, for example to make behavior more consistent in an edge case, match what is implemented by  the large majority of actual implementations, clarify wording, or occasionally add a minor feature. For bigger changes, use the feature request process described below.
```

Comment:
Given that the definitions of "minor feature" and "bigger changes" are subjective it might be best to just leave out this possibility for the sake of clarity.

Reply:
@littledan Sorry, what I said was pretty misleading... we def want to say that PRs are for minor things only. Above I meant more omitting the possibility of "minor features" landing via PRs rather than the whole paragraph. So, something like:

> Issues and PRs in the ecma262 repository are appropriate for minor modifications to the existing specification, for example to make behavior more consistent in an edge case, match what is implemented by  the large majority of actual implementations, or to clarify wording. New features use the feature request process described below.

Reply:
Applied!